lib/reporting/proofing_rate_report.rb

```diff
@@ -52,20 +54,20 @@ def as_csv
 
       csv << ['Metric', *DATE_INTERVALS.map { |days| "Trailing #{days}d" }]
 
-      csv << ['Start Date', *reports.map(&:time_range).map(&:begin).map(&:to_date)]
-      csv << ['End Date', *reports.map(&:time_range).map(&:end).map(&:to_date)]
+      csv << ['Start Date', *reduced_reports.map(&:time_range).map(&:begin).map(&:to_date)]
+      csv << ['End Date', *reduced_reports.map(&:time_range).map(&:end).map(&:to_date)]
 
-      csv << ['IDV Started', *reports.map(&:idv_started)]
-      csv << ['Welcome Submitted', *reports.map(&:idv_doc_auth_welcome_submitted)]
-      csv << ['Image Submitted', *reports.map(&:idv_doc_auth_image_vendor_submitted)]
-      csv << ['Successfully Verified', *reports.map(&:successfully_verified_users)]
-      csv << ['IDV Rejected (Non-Fraud)', *reports.map(&:idv_doc_auth_rejected)]
-      csv << ['IDV Rejected (Fraud)', *reports.map(&:idv_fraud_rejected)]
+      csv << ['IDV Started', *reduced_reports.map(&:idv_started)]
+      csv << ['Welcome Submitted', *reduced_reports.map(&:idv_doc_auth_welcome_submitted)]
+      csv << ['Image Submitted', *reduced_reports.map(&:idv_doc_auth_image_vendor_submitted)]
+      csv << ['Successfully Verified', *reduced_reports.map(&:successfully_verified_users)]
+      csv << ['IDV Rejected (Non-Fraud)', *reduced_reports.map(&:idv_doc_auth_rejected)]
+      csv << ['IDV Rejected (Fraud)', *reduced_reports.map(&:idv_fraud_rejected)]
 
-      csv << ['Blanket Proofing Rate (IDV Started to Successfully Verified)', *reports.map(&:blanket_proofing_rates)]
-      csv << ['Intent Proofing Rate (Welcome Submitted to Successfully Verified)', *reports.map(&:intent_proofing_rates)]
-      csv << ['Actual Proofing Rate (Image Submitted to Successfully Verified)', *reports.map(&:actual_proofing_rates)]
-      csv << ['Industry Proofing Rate (Verified minus IDV Rejected)', *reports.map(&:industry_proofing_rates)]
+      csv << ['Blanket Proofing Rate (IDV Started to Successfully Verified)', *reduced_reports.map(&:blanket_proofing_rates)]
+      csv << ['Intent Proofing Rate (Welcome Submitted to Successfully Verified)', *reduced_reports.map(&:intent_proofing_rates)]
```

Comment:
With these lines getting kinda long, I wonder if it starts to make sense to make them all multiline? Or just where they're too long?

```suggestion
      csv << [
        'Intent Proofing Rate (Welcome Submitted to Successfully Verified)',  
        *reduced_reports.map(&:intent_proofing_rates)
      ]
```

Reply:
The whole thing is already wrapped in `# rubocop:disable Layout/LineLength` so I'm not going to worry about this.